inference-engine/thirdparty/clDNN/kernel_selector/core/common/kernel_selector_utils.cpp

```diff
@@ -220,35 +220,44 @@ std::vector<size_t> GetOptimalLocalWorkGroupSizes(std::vector<size_t> gws, const
 bool CheckInputsOutputNoPitchSameDims(const base_params& params) {
     bool no_pitch_same_dims = true;
 
+    std::vector<std::pair <DataLayout, int>> block_layouts = {
+        {DataLayout::b_fs_yx_fsv16,          16},
+        {DataLayout::b_fs_zyx_fsv16,         16},
+        {DataLayout::b_fs_yx_fsv32,          32},
+        {DataLayout::b_fs_zyx_fsv32,         32},
+        {DataLayout::bs_fs_yx_bsv16_fsv16,   16},
+        {DataLayout::bs_fs_zyx_bsv16_fsv16,  16},
+        {DataLayout::bs_f_bsv8__af8,         8},
+        {DataLayout::bs_f_bsv16__af8,        16},
+        {DataLayout::b_fs_yx_fsv4,           4},
+        {DataLayout::fs_b_yx_fsv32,          32},
+        {DataLayout::b_fs_yx_32fp,           32}
+    };
```

Comment:
Looks like it is better to use std::map for this purpose
BTW what about double blocked format e.g. bs_fs_yx_bsv16_fsv16?